client/src/utils/string.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Capitalise a string
+ *
+ * @param {string} string String to capitalise
+ * @returns {string} Capitalised string
+ */
+export function capitalise(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+/**
+ * Convert a camel cased string ('fooBar') to start case ('Foo Bar')
+ *
+ * @param {string} string String to convert
+ * @returns {string} Start case string
+ */
+export function camelCaseToStartCase(string) {
+  const split = string.replace(/([A-Z])/g, ' $1');
```

Comment:
Would be good to have unit tests created for these util functions in another PR.

Reply:
Added as a board card/entry/whatever-it's-meant-to-be-called
https://www.notion.so/Write-unit-tests-for-DAShboard-client-util-functions-870703bfb08d4dea800511e387e13b8a